### DISCUSSION TITLE
How to run the project?

### DISCUSSION THREAD

**Discussion by vaibhavs2** [syedibrahim/Pharmacy-Management-System#5] [Q&A] [answered]
Please provide some details about how to run this project?

**Comment by syedibrahim** [comment, marked as answer]
1. Install Eclipse J2EE or any other IDE of your choice.
2. Use WAMP or XAMPP or any server of your choice.
3. First create the MySql database on WAMP server using the sql file in the repo 
4. And then build and run the project from the IDE.
